web/src/javascript/game.js

```diff
@@ -0,0 +1,58 @@
+import {
+  mapWidth,
+  mapHeight,
+  drawCellsOnMap,
+  drawMap,
+  drawMiniMap,
+} from "./drawMap";
+
+const points = [];
+const numPoints = 50;
+
+function extractPointsFromState(gameState) {
+  return [].concat.apply(...gameState.players.map((p) => p.cells));
+}
+
+export function draw(gameState) {
+  drawCellsOnMap(extractPointsFromState(gameState));
+  drawMap();
+  drawMiniMap();
+}
+
+export function update(gameState) {
+  let i;
+  let point;
+  let len = points.length;
+  for(i = 0; i < len; i += 1) {
+    point = points[i];
+    point.x += point.vx;
+    point.y += point.vy;
+    if(point.x > mapWidth ||
+       point.x < 0 ||
+       point.y > mapHeight ||
+       point.y < 0) {
+      initPoint(point);
+    }
+  }
+}
+
+//Temporary function to add cells
+//This function will be replaced by real cells
+function initPoint(p) {
```

Comment:
I'll remove the fake cells in the next PR. I tried to touch the minimum required.